include/gatekeeper_gt.h

```diff
@@ -59,10 +66,28 @@ struct gt_config {
 	uint16_t           ggu_src_port;
 	uint16_t           ggu_dst_port;
 
+	/* The maximum number of neighbor entries for the GT. */
+	int                max_num_ipv6_neighbors;
+
 	/*
 	 * The fields below are for internal use.
 	 * Configuration files should not refer to them.
 	 */
+
+	/*
+	 * Specify the network prefixes that the GT block responsible for.
+	 * There may not be the IPv4 or IPv6 address.
+	 */
+	uint8_t            configured_proto;
+
+	/* IPv4 address and network mask. */
+	struct in_addr     ip4_addr;
+	int8_t             ip4_addr_plen;
```

Comment:
I don't think we ever really use `ip4_addr`, so why include it? It seems like we only need to know the IPv4 subnet prefix length to set the size of the table correctly, but I think we can get that from the network information.